package.json

```diff
@@ -34,6 +35,7 @@
     "immutable": "^3.8.1",
     "jsonwebtoken": "^7.4.1",
     "knex": "^0.13.0",
+    "lodash": "^4.17.4",
```

Comment:
Yeah, I added it bc I though it might be nice to use if we want to parse data coming from the server